### PR TITLE
ci: drop bors from linux.yml

### PR DESCRIPTION
**Pull request by Saviq**

### CODE REVIEW

**Comment by Saviq** [comment]
@townsend2010 you will need to review-approve this. Also, since I pushed, it needs "Merge when ready" again.

File: .github/workflows/linux.yml
Comment (Saviq) [inline .github/workflows/linux.yml:258]: @townsend2010 you'll want to drop the bare "Linux" check from required checks. This was only here so we needn't list all the build types in `bors.toml`.

Comment (townsend2010) [reply .github/workflows/linux.yml:258]: Hey @Saviq, I don't think the bare Linux check is a required check from what I can tell.

**Review by townsend2010 — DISMISSED** [review]
Hey @Saviq!

Cool, thanks!  This makes sense to me.

**Comment by Saviq** [comment]
@townsend2010 probably want to add Lint, though:

**Comment by townsend2010** [comment]
@Saviq Yep, thanks, added that and cla-check as well.

**Comment by Saviq** [comment]
Huuh?

```
 -block-labels = [ "no-merge" ]
Error:  Forbidden string(s) found in the diff ^ (matched "(?i)\bno[_-]?merge\b")
```

@townsend2010 you may want to drop Lint for now after all :P

**Comment by townsend2010** [comment]
Strange, I don't even understand what it's complaining about here, but ok, whatever, I will remove it for now and enable it again after this is merged.

**Comment by Saviq** [comment]
> I don't even understand what it's complaining about here

Yeah, about the file that I just deleted ¯\_(ツ)_/¯

**Comment by ricab** [comment]
Hi guys, where did that show up? I think it is just noticing the "forbidden string" in the diff and complaining because of that. It should probably only do so for added lines...

**Comment by ricab** [comment]
`bors.toml` would normally have been excluded, but that is also changing with this diff. Do you mind if I commit something on top for fix the "no-merge" issue here?

**Comment by Saviq** [comment]
Please don't commit here. I think we should just ignore Lint on this one.

**Comment by ricab** [comment]
You're removing the exclusion for bors.toml, I thought that could justify replacing it with a more generic exclusion. But OK, I will make a separate PR.

**Review by townsend2010 — DISMISSED** [review]
Let's go for this again!

**Review by townsend2010 — DISMISSED** [review]
LGTM, thanks!

**Review by townsend2010 — DISMISSED** [review]
Let's try this

**Review by townsend2010 — APPROVED** [review]
Ok, LGTM.  Will merge manually after the CI run.

**Review by townsend2010 — APPROVED** [review]
:+1:

**Comment by townsend2010** [comment]
Ok, seems this is ok and will merge manually.

**Pull request merged**
